frontend/src/lib/components/ProfilePicture/ProfilePicture.scss

```diff
@@ -53,6 +53,9 @@
 .ProfileBubbles {
     display: flex;
     align-items: center;
+    > * {
```

Comment:
Nitpick but wildcard selectors are pretty inefficient in terms of CSS rendering. If can specify an element or selector here specifically we'll see faster rendering. Not a blocker, just noticed it so figured I'd mention.

Reply:
That's fair!